pkgs/development/tools/nimble/default.nix

```diff
@@ -1,4 +1,4 @@
-{ stdenv, fetchFromGitHub, nim, openssl, patchelf }:
+{ stdenv, fetchFromGitHub, nim, openssl, patchelf, glibc }:
```

Comment:
Both `patchelf` and `glibc` are implicit in the standard builder environment. (In general, it is probably safer to refer to libc via `stdenv.cc.libc`).